blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,260 @@
+
+## Improved campaign creation, management, and syncing performance
+
+TODO: Video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+The overall experience of generating diffs, creating and reviewing a campaign, then managing the changes to closure has been improved.
+
+Using the `src-cli` to generate diffs and create a campaign plan has been improved:
+
+* It is now easier to create a campaign plan for successful patches when other patches fail for some repositories.
+* The dependencies on the external diff and unzip tools have been removed, instead git is used to produce diffs.
+* .gitignore files are now respected when creating patches.
+* On macOS src CLI can [now be installed with Homebrew.](https://github.com/sourcegraph/homebrew-src-cli)
+
+Take a look at the [improved documentation](https://docs.sourcegraph.com/user/campaigns) for usage and examples.
+
+When creating your campaign in Sourcegraph, it is now clearer what state the campaign is in and is easier to differentiate between draft patches and changesets that have been created on the code host. Additionally, you can now filter the changesets to quickly find which ones need follow up.
+
+Behind the scenes, campaign changesets are now kept up to date using a heuristic solution that reduces the load placed on the code host. Previously, each changeset was updated during regularly scheduled update processes. The new implementation prioritizes syncing changesets based on when last they changed and is also aware of real-time changes that arrive via webhooks.
+
+Code change management campaigns are in private beta. [Watch the campaigns screencasts](https://about.sourcegraph.com/product/code-change-management#see-it-in-action) to see what we have planned, and [apply for early access](https://about.sourcegraph.com/contact/request-code-change-management-demo/) to campaigns for your organization.
+
+## Required migration for Kubernetes deployments
+
+All admins of existing instances with Kubernetes deployments will need to follow the [migration guide for 3.14](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md#314-unreleased). This migration changes the container user from root to non-root for all containers. This change makes Sourcegraph easier to deploy for organizations with strict security requirements. Docker containers and docker-compose deployments are not affected.
+
+## Improved debuggability of Kubernetes deployments
```

Comment:
When I read the header at the top of the page, I wasn't sure what it was referring to (especially because of the monitoring improvements in the bullet immediately beneath it, which felt a bit overlapping). What do you think about "Profiling and tracing for Kubernetes deployments"? 

If the goal is to focus on the use case, rather than the technical change/fix, then I feel like we should change other bullets as well to match this. And still perhaps this one could be something like:

"Easier debugging of Kubernetes deployments with profiling and tracing"

I leave it up to you and @uwedeportivo on this though!